lib/modules/commandLine.js

```diff
@@ -84,16 +84,19 @@ const commandLine = _.once(() => {
 		}
 	});
 
-	input.addEventListener('keyup', (e: KeyboardEvent) => {
+	document.addEventListener('keyup', (e: KeyboardEvent) => {
 		if (e.key === NAMED_KEYS.Escape) {
 			// close prompt.
 			toggleCmdLine(false);
-		} else {
-			// auto suggest?
-			cmdLineHelper($(e.target).val());
+			e.stopImmediatePropagation();
```

Comment:
Hmm, this listener never gets removed, so this might prevent other "press escape to close" things from working?

Reply:
Yeah, I've overlooked that. That must be fixed.